plc4j/api/src/test/java/org/apache/plc4x/java/api/connection/PlcReaderTest.java

```diff
@@ -38,24 +36,24 @@ Licensed to the Apache Software Foundation (ASF) under one
 
 public class PlcReaderTest {
 
-    @Test
+/*    @Test
```

Comment:
I don't get the whole outcommenting of tests right now. It this then an open TODO or what is the reason for this?

Reply:
Well the API is currently just interfaces ... how do you test them? ... and that's why there's a WIP in the Pull-Request as I will probably move these to driver-base where the real datatypes will be. I commented them out so I can compile the parts I have migrated and don't have to fetch them from the history when re-adding them somewhere else.

Reply:
ok, fair enough :)